deployment/debian-package-x64/Dockerfile

```diff
@@ -1,23 +1,40 @@
 FROM debian:9
```

Comment:
Would be simpler to use the official Dotnet stretch image.
https://github.com/dotnet/dotnet-docker/blob/master/TAGS

Looks like it does roughly the same thing. 
https://github.com/dotnet/dotnet-docker/blob/master/2.2/sdk/stretch/amd64/Dockerfile

Reply:
Didn't realize those existed - that would allow us to remove most of the build stages yes?

Reply:
Yup!

Reply:
Done!